app/graphql/types/mutation_fields/systemnotification.rb

```diff
@@ -0,0 +1,22 @@
+# app/graphql/types/mutation_fields/system_notification_mutation_fields.rb
+module Types
+  module MutationFields
+    module SystemNotificationMutationFields
+      extend ActiveSupport::Concern
+
+      included do
```

Comment:
We need a delete mutation

Also a mutation `MarkSystemNotificationsAsRead` where you receive a bunch of uuids of system notifications and you can mark them as read